test/dotnet-help.Tests/GivenThatIWantToShowHelpForDotnetCommand.cs

```diff
@@ -65,5 +66,39 @@ public void WhenHelpOptionIsPassedToDotnetItPrintsUsage(string helpArg)
             cmd.Should().Pass();
             cmd.StdOut.Should().ContainVisuallySameFragment(HelpText);
         }
+
+        [Fact]
+        public void WhenInvalidCommandIsPassedToDOtnetHelpItPrintsError()
+        {
+          var cmd = new DotnetCommand()
+                .ExecuteWithCapturedOutput("help invalid");
+          
+          cmd.Should().Fail();
+          cmd.StdErr.Should().ContainVisuallySameFragment($"Specified command invalid is not a valid CLI command. Please specify a valid CLI commands. For more information, run dotnet help.");
```

Comment:
Also, "...please specify a valid CLI commands" seems wrong.

Should we just go ahead and print the commands here?